edge/pkg/edged/edged.go

```diff
@@ -491,3 +491,29 @@ func (e *edged) controllerUnpublishVolume(content []byte) (interface{}, error) {
 func filterPodByNodeName(pod *v1.Pod, nodeName string) bool {
 	return pod.Spec.NodeName == nodeName
 }
+
+func kubeletHealthCheck(port int32, kubeletReadyChan chan struct{}) {
+	url := fmt.Sprintf("http://localhost:%d/healthz/syncloop", port)
```

Comment:
Plz add more description for the PR that how can we guarantee that kubelet is ready to sync pods after getting 200 from `healthz/syncloop`,  thanks.